modules/call-dispatch/src/lib.rs

```diff
@@ -43,14 +50,52 @@ pub type SpecVersion = u32;
 /// Weight of single deposit_event() call.
 const DEPOSIT_EVENT_WEIGHT: Weight = 0;
 
+/// Origin of the call on the target chain.
+#[derive(RuntimeDebug, Encode, Decode, Clone)]
+pub enum CallOrigin<SourceChainAccountPublic, TargetChainAccountPublic, TargetChainSignature> {
+	/// Call is originated from bridge account, which is (designed to be) specific to
+	/// the single deployed instance of the messages bridge (message-lane, ...) module.
+	/// It is assumed that this account is not controlled by anyone and has zero balance
+	/// (unless someone would make transfer by mistake?).
```

Comment:
```suggestion
	/// (unless someone would make transfer by mistake?).
	/// If we trust the source chain to allow sending calls with that origin in case they originate from source chain `root` account (default implementation),
	/// `BridgeAccount` represents the source-chain-root origin on the target chain and can be used to send and authorize
	/// "control plane" messages between the two runtimes. 
```